examples/bsc-example/src/app/bsc.nft.example.ts

```diff
@@ -78,17 +78,16 @@ export async function bscNftExample() {
 
   const minted = await bscSDK.nft.mintNFT({
     chain: 'BSC',
-    tokenId: '100000',
-    to: '0x687422eEA2cB73B5d3e242bA5456b782919AFc85',
-    erc20: '0x687422eEA2cB73B5d3e242bA5456b782919AFc85',
-    contractAddress: '0x687422eEA2cB73B5d3e242bA5456b782919AFc85',
+    to: '0x811DfbFF13ADFBC3Cf653dCc373C03616D3471c9',
```

Comment:
tokenId and contractAddress must be present as well as fromPrivateKey

Reply:
Added back, thought for the MintNftExpress it is sent without these.

Reply:
So prepare another example for NFT express